Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -2105,7 +2105,7 @@ NativeArray<byte> outputStream
         {
             var node = new Node
             {
-                name = name,
+                name = EscapeJson(name),
```

Comment:
The actual problem is not the name of nodes/meshes containing certain special characters, but that they are not serialized properly to JSON.

So the problem should be solved in `JsonWriter`, which has the benefit of fixing it for any string literal.